### ISSUE TITLE
crash when crafting atlas with shears

### ISSUE THREAD

**Issue by LeoBeliik** [MehVahdJukaar/mapatlases-neoforge#91]
crash on server, dunno client

https://mclo.gs/OVmKtYY

vm modpack (SuperQuarkPackSPE)

edit: it only crashes in the twilight forest, or at least, not in the overworld

**Comment by MehVahdJukaar** [comment]
ehm did you try ty remove maps from an empty atlas?

**Comment by LeoBeliik** [comment]
No. The atlas had a few maps in, both for the twilight forest and the overworld
